web/translation/translate.ru_RU.toml

```
@@ -89,33 +89,33 @@
 "xrayStatus" = "Xray"
 "stopXray" = "Остановить"
 "restartXray" = "Перезапустить"
-"xraySwitch" = "Версия"
+"xraySwitch" = "Выбор версии"
 "xraySwitchClick" = "Выберите желаемую версию"
-"xraySwitchClickDesk" = "Выбирайте внимательно, так как старые версии могут быть несовместимы с текущими конфигурациями"
+"xraySwitchClickDesk" = "Обратите внимание: старые версии могут не поддерживать текущие настройки"
 "operationHours" = "Время работы системы"
 "systemLoad" = "Системная нагрузка"
```

```suggestion
"systemLoad" = "Нагрузка на систему"
```

96. "Системная нагрузка" -> "Нагрузка на систему"

Предлагаю этот вариант, так как он звучит естественнее

---

Обновите перевод с учетом комментариев и вашего личного мнения